utils/parenthesesutils.go

```diff
@@ -55,12 +55,26 @@ func RemovePlaceholderParentheses(pattern, target string) string {
 	return temp
 }
 
-// Escaping Parentheses with no corresponding placeholder.
-func addEscapingParentheses(pattern, target string) string {
+// AddEscapingParenthesesForUploadCmd escaping parentheses with no corresponding placeholder.
```

Comment:
escaping --> escapes

The same goea for the below `AddEscapingParentheses` function.